gallery/gallery/lib/demos/cupertino/cupertino_slider_demo.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN cupertinoSliderDemo
+
+class CupertinoSliderDemo extends StatefulWidget {
+  @override
+  _CupertinoSliderDemoState createState() => _CupertinoSliderDemoState();
+}
+
+class _CupertinoSliderDemoState extends State<CupertinoSliderDemo> {
+  double _value = 25.0;
+  double _discreteValue = 20.0;
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: CupertinoNavigationBar(
+        middle: Text(GalleryLocalizations.of(context).demoCupertinoSliderTitle),
+      ),
+      child: DefaultTextStyle(
+        style: CupertinoTheme.of(context).textTheme.textStyle,
+        child: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              Column(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  SizedBox(
+                    width: double.infinity,
+                    child: CupertinoSlider(
+                      value: _value,
+                      min: 0.0,
+                      max: 100.0,
+                      onChanged: (value) {
+                        setState(() {
+                          _value = value;
+                        });
+                      },
+                    ),
+                  ),
+                  MergeSemantics(
+                    child: Text(
+                      GalleryLocalizations.of(context)
+                          .demoCupertinoSliderContinuous(
+                        _value.toStringAsFixed(1),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+              Column(
+                mainAxisSize: MainAxisSize.min,
+                children: <Widget>[
```

Comment:
nit: Don't need the `<Widget>` here. As long as you have different kinds of widget in the list, Dart will assume a type of `List<Widget>`.

Reply:
Done, I think I just missed this one.